prac_06/programming_language.py

```diff
@@ -0,0 +1,19 @@
+"""Programming Language Class"""
+
+
+class ProgrammingLanguage:
+    def __init__(self, name, typing, reflection, year):
```

Comment:
When initializing remember to give default values. For example:
def __init__(self, name="", typing="", reflection="", year=0):